utils/run-test

```diff
@@ -171,6 +171,7 @@ def main():
         # build_dir.
         paths = args.paths
         if not paths:
+            parser.print_usage()
             error_exit("error: too few arguments")
```

Comment:
@dabrahams Since this command requires at least one argument, I'm considering no arguments to be invalid. Okay to merge?